zeroex/orderwatch/order_watcher_test.go

```diff
@@ -1353,25 +1174,37 @@ func TestOrderWatcherHandleOrderExpirationsUnexpired(t *testing.T) {
 	signedOrders := scenario.NewSignedTestOrdersBatch(t, 2, orderOptions)
 	signedOrderOne := signedOrders[0]
 	signedOrderTwo := signedOrders[1]
-	blockwatcher, orderWatcher := setupOrderWatcher(ctx, t, ethRPCClient, meshDB)
+	blockwatcher, orderWatcher := setupOrderWatcher(ctx, t, ethRPCClient, database)
 	watchOrder(ctx, t, orderWatcher, blockwatcher, ethClient, signedOrderOne)
 	watchOrder(ctx, t, orderWatcher, blockwatcher, ethClient, signedOrderTwo)
 
 	orderEventsChan := make(chan []*zeroex.OrderEvent, 2*orderWatcher.maxOrders)
 	orderWatcher.Subscribe(orderEventsChan)
 
 	// Simulate a block found with a timestamp past expirationTime
-	latestBlock, err := meshDB.FindLatestMiniHeader()
+	latestBlocks, err := database.FindMiniHeaders(&db.MiniHeaderQuery{
+		Limit: 1,
+		Sort: []db.MiniHeaderSort{
+			{
+				Field:     db.MFNumber,
+				Direction: db.Descending,
+			},
+		},
+	})
```

Comment:
This is another example of why the utility method would be nice.

Reply:
There are several more throughout the codebase. I'd suggest that we just grep for this regex or even just grep for `FindMiniHeaders`.

Reply:
👍 Good idea. I added a helper method and a test for it.